extensions/ql-vscode/src/view/model-editor/MethodName.tsx

```diff
@@ -4,6 +4,7 @@ import { Method } from "../../model-editor/method";
 
 const Name = styled.span`
   font-family: var(--vscode-editor-font-family);
+  word-break: break-all;
```

Comment:
I had to add this otherwise it was breaking the grid at low screen widths. I'm not sure why this is needed now and wasn't before.